Makefile

```diff
@@ -55,31 +48,35 @@ SYSCONFDIR:=$(DEFAULT_SYSCONFDIR)
 endif
 SYSCONFDIR:=$(subst PREFIX,$(PREFIX),$(SYSCONFDIR))
 
-export APPLY_PATCHES
+ifndef NEW_BUILD
+NEW_BUILD=$(DEFAULT_NEW_BUILD)
+endif
+
 export VERSION
 export DESTDIR
 export PREFIX
 export SYSCONFDIR
 export SKIP_TESTS
+export NEW_BUILD
 
 all: install
 
 
 archive: install
+ifeq ($(NEW_BUILD),no)
 	rm -f $(CURDIR)/$(PACKAGE_NAME).tar.gz && cd $(DESTDIR) && tar -czf $(CURDIR)/$(PACKAGE_NAME).tar.gz $(PREFIX)
 	rm -f $(CURDIR)/$(PACKAGE_NAME).zip && cd $(DESTDIR) && zip -r $(CURDIR)/$(PACKAGE_NAME).zip $(PREFIX)
-
-apply-patches: $(wildcard patches/*)
-ifeq ($(APPLY_PATCHES),yes)
-	git reset --hard HEAD
-	cat patches/series | xargs -iPATCH bash -c 'patch -p1 < patches/PATCH'
 endif
 
-build: apply-patches
+build:
+ifeq ($(NEW_BUILD),yes)
+	pip3 install requests
```

Comment:
we should be fine having this installed generally, so this would be better moved to the code that generates the AMIs https://github.com/confluentinc/confluent-amis so it is available in all builds. Otherwise all the component repos will have to duplicate this code to install `requests`